### PR TITLE
Upstream Stuff

### PR DESCRIPTION
**Pull request by glowredman**
**From Upstream:**
- Speed boost (Infinity Boots) can be disabled via config, default is enabled
- Added Clay Singularity (for ultimate balance!)
- Added Dire Crafting Table recipe for Infinity Egg (Witchery)
- Fix removal of Compressor recipes (MineTweaker)
- Changed Infinity and Neutronium TiCon stats
- Added NBT handling for Dire Crafting Table recipes
- Added Starfuel and Neutronium Gear (both currently useless, as the Dire Autocrafting Table was never finished)
- Removed Morv-In-A-Box (was not registered anyway)
- Added German localisation
- Removed Deadrock textures
- Updated textures for Infinity Axe, Infinity Destroyer, some Singularities

**Extra:**
- Update buildscript

Infinity Egg:
grafik

Star Fuel:
grafik

Neutronium Gear:
grafik

### CODE REVIEW

File: src/main/java/fox/spiteful/avaritia/blocks/LudicrousBlocks.java [hunk 1]
```diff
@@ -21,6 +19,7 @@
      */
     public static Block resource_block;
     public static Block dire_crafting;
+    public static Block auto_dire_crafting;
```
Comment (miozune) [inline src/main/java/fox/spiteful/avaritia/blocks/LudicrousBlocks.java:22]: oversight?

File: src/main/resources/assets/avaritia/lang/de_DE.lang [hunk 2]
```diff
@@ -0,0 +1,183 @@
...
+#
+# Blocks
+tile.compressed_workbench.name=Komprimierte Werkbank
+tile.very_compressed_workbench.name=Doppelt komprimierte Werkbank
+tile.block_crystal_matrix.name=Kristallmatrix
+tile.block_neutronium.name=Neutronenblock
+tile.block_infinity.name=Unendlichkeitsblock
+tile.block_gaia.name=Gaia Block
+tile.dire_crafting.name=Schreckliche Werkbank
+tile.auto_dire_crafting.name=Schreckliche Werkbank MK 20
```
Comment (miozune) [inline src/main/resources/assets/avaritia/lang/de_DE.lang:61]: same

File: src/main/resources/assets/avaritia/lang/en_US.lang [hunk 3]
```diff
@@ -53,6 +56,7 @@ tile.block_neutronium.name=Neutronium Block
 tile.block_infinity.name=Infinity Block
 tile.block_gaia.name=Gaia Block
 tile.dire_crafting.name=Dire Crafting Table
+tile.auto_dire_crafting.name=Dire Crafting Table Mk XX
```
Comment (miozune) [inline src/main/resources/assets/avaritia/lang/en_US.lang:59]: same

**Review by GTNH-Colen — APPROVED** [review]

**Comment by GTNH-Colen** [comment]
Resolves https://github.com/GTNewHorizons/GT-New-Horizons-Modpack/issues/5897 at least.

**Pull request merged**